keras_rs/src/layers/embedding/distributed_embedding_test.py

```diff
@@ -318,6 +319,37 @@ def test_model_fit(self, input_type, use_weights):
         with self._strategy.scope():
             layer = distributed_embedding.DistributedEmbedding(feature_configs)
 
+        # Mark ragged inputs as sparse in JAX for the purposes
+        # of preprocessing.  Trying to create a "ragged=True" keras.Input for
```

Comment:
Well, should we just change that?

Reply:
I thought about it - Keras throws an exception for good reason: the backend is marked as not supporting Ragged.  It does work if I remove that check from `keras.KerasTensor`.